minio/minio
# helm: apply templating to Ingress host and environment variable values

## Description
Pass select Helm values through the `tpl` template function so they are rendered as template code.

## Motivation and Context
Passing certain values through the `tpl` function can help improve reusability by enabling to use parameters such as the Helm release name when rendering manifests. For example, we can create a configuration including values such as:

```yaml
environment:
  MINIO_BROWSER_REDIRECT_URL: "http://{{ printf \"%s-console\" (include \"minio.fullname\" .) }}.example.com"
ingress:
  hosts:
  - "{{ include \"minio.fullname\" . }}.example.com"
consoleIngress:
  hosts:
  - "{{ printf \"%s-console\" (include \"minio.fullname\" .) }}.example.com"
```

## How to test this PR?
Install the chart with values of the form of those suggested above. Verify that Ingresses are created with names based on the `minio.fullname` template.

## Types of changes

## Checklist:
